notifier/notifierRunner.go

```diff
@@ -0,0 +1,119 @@
+package notifier
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/notifier-go/api/gin"
+	"github.com/ElrondNetwork/notifier-go/api/shared"
+	"github.com/ElrondNetwork/notifier-go/config"
+	"github.com/ElrondNetwork/notifier-go/dispatcher"
+	"github.com/ElrondNetwork/notifier-go/facade"
+	"github.com/ElrondNetwork/notifier-go/factory"
+	"github.com/ElrondNetwork/notifier-go/rabbitmq"
+)
+
+var log = logger.GetOrCreate("notifierRunner")
+
+var (
+	backgroundContextTimeout = 5 * time.Second
```

Comment:
this is not background, might rename it to contextTimeout

Reply:
this is not being used actually, forgot to remove after refactoring.
removed it now